cat: filename_temp.txt: No such file or directory

## Bug Description:
Steps to reproduce:
1. **New**  `git clone git@github.com:quipucords/yupana.git`
2. Run `make sample-data`

Error:
```cat: filename_temp.txt: No such file or directory
cat: filename_temp.txt: No such file or directory
mkdir -p temp/reports
mkdir -p temp/old_reports_temp
tar -xvzf sample.tar.gz -C temp/old_reports_temp
x 2dd60c11-ee5b-4ddc-8b75-d8d34de86a34.json
x eb45725b-165a-44d9-ad28-c531e3a1d9ac.json
x metadata.json
python scripts/change_uuids.py
2dd60c11-ee5b-4ddc-8b75-d8d34de86a34.json
eb45725b-165a-44d9-ad28-c531e3a1d9ac.json
metadata.json
tar: no files or directories specified
make: *** [sample-data] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -113,26 +113,26 @@ sample-data:
 	mkdir -p temp/old_reports_temp
 	tar -xvzf sample.tar.gz -C temp/old_reports_temp
 	python scripts/change_uuids.py
-	@echo sample_data_ready_$(shell date +%s).tar.gz > filename_temp.txt
-	@cd temp; COPYFILE_DISABLE=1 tar -zcvf ${shell cat filename_temp.txt} reports
-	rm -rf temp/reports
-	rm -rf temp/old_reports_temp
-	@echo ""
-	@echo "The updated report was written to" temp/${shell cat filename_temp.txt}
-	@echo ""
+	@NEW_FILENAME="sample_data_ready_$(shell date +%s).tar.gz"; \
+	cd temp; COPYFILE_DISABLE=1 tar -zcvf $$NEW_FILENAME reports; \
+	echo ""; \
+	echo "The updated report was written to" temp/$$NEW_FILENAME; \
+	echo ""; \
+	rm -rf reports; \
+	rm -rf old_reports_temp
 
 custom-data:
 	mkdir -p temp/reports
 	mkdir -p temp/old_reports_temp
 	tar -xvzf $(file) -C temp/old_reports_temp
 	python scripts/change_uuids.py
-	@echo sample_data_ready_$(shell date +%s).tar.gz > filename_temp.txt
-	@cd temp; COPYFILE_DISABLE=1 tar -zcvf ${shell cat filename_temp.txt} reports
-	rm -rf temp/reports
-	rm -rf temp/old_reports_temp
-	@echo ""
-	@echo "The updated report was written to" temp/${shell cat filename_temp.txt}
-	@echo ""
+	@NEW_FILENAME="sample_data_ready_$(shell date +%s).tar.gz"; \
+	cd temp; COPYFILE_DISABLE=1 tar -zcvf $$NEW_FILENAME reports; \
+	echo ""; \
+	echo "The updated report was written to" temp/$$NEW_FILENAME; \
+	echo ""; \
+	rm -rf reports; \
+	rm -rf old_reports_temp
 
 upload-data:
 	curl -vvvv -H "x-rh-identity: $(shell echo '{"identity": {"account_number": $(RH_ACCOUNT_NUMBER), "internal": {"org_id": $(RH_ORG_ID)}}}' | base64)" \

